.github/workflows/e2e-tests.yml

```
@@ -165,18 +167,20 @@ jobs:
         uses: pnpm/action-setup@v4
         with:
           run_install: false
-      - uses: useblacksmith/setup-node@v5
+      - uses: actions/setup-node@v5
         with:
           node-version: ${{ matrix.node-version }}
```

actionlint (1.7.7)

property "node-version" is not defined in object type {test-suite: any}

(expression)